components/Tabs/src/MDCTabBarIndicatorTemplate.h

```diff
@@ -0,0 +1,36 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Please update copyright headers here and throughout.

Reply:
Done